shared/src/main/scala/amf/client/remod/AMFRender.scala

```diff
@@ -5,8 +5,8 @@ import org.yaml.model.YDocument
 
 private[remod] object AMFRender {
 
-  def render(bu:BaseUnit, env:BaseEnvironment): String = ???
+  def render(bu:BaseUnit, env: AMFConfiguration): String = ???
```

Comment:
Looking into this interface i think it will have to at least receive the target vendor, how would conversion be handled? Media type could possibly be added too, or we can just define a default mediatype for each vendor.

Reply:
I think that conversation required a dipper analysis (that you are doing) and a discussion based on that analysis.
Using this channel to talk about this could be very complex.

Reply:
Added medita type as parameter (following the vendor idea)
The idea of the methods that serialized only on base unit was to serialize the to the origin vendor, or graph if is not present.